sqlcapture/interface.go

```diff
@@ -73,7 +72,7 @@ type ChangeEvent struct {
 // FlushEvent informs the generic sqlcapture logic about transaction
 // boundaries.
 type FlushEvent struct {
-	Source SourceMetadata
+	Cursor string // TODO(wgd): Maybe json.RawMessage?
```

Comment:
Not sure if this is a WIP todo or something for a future commit. With the way I'm reading this PR it seems like a string worked pretty well.

Reply:
Removed